internal/cmd/restorer.go

```diff
@@ -225,6 +228,13 @@ func (r *restorer) commitStream(ctx context.Context, bulkImportClient v1.Experim
 	canceled, cancelErr := isCanceledError(ctx.Err(), err)
 	unknown := !retryable && !conflict && !canceled && err != nil
 
+	intExpectedLoaded, castErr := safecast.ToInt64(expectedLoaded)
```

Comment:
I'll give this a go. I assumed that the value coming out of the DB lib was an int rather than a uint, so it might just move the casting issue around, but I agree that would be nice.